rest/model/admin.go

```diff
@@ -363,13 +364,24 @@ func (a *APIAmboyConfig) BuildFromService(h interface{}) error {
 		a.GroupTTLMinutes = v.GroupTTLMinutes
 		a.RequireRemotePriority = v.RequireRemotePriority
 		a.LockTimeoutMinutes = v.LockTimeoutMinutes
+		if err := a.Retry.BuildFromService(v.Retry); err != nil {
+			return errors.Wrap(err, "building Amboy retry settings from service")
+		}
 	default:
 		return errors.Errorf("%T is not a supported type", h)
 	}
 	return nil
 }
 
 func (a *APIAmboyConfig) ToService() (interface{}, error) {
+	i, err := a.Retry.ToService()
+	if err != nil {
+		return nil, errors.Wrap(err, "converting Amboy retry settings to service")
+	}
+	retry, ok := i.(evergreen.AmboyRetryConfig)
+	if !ok {
```

Comment:
Do we really need to check `ok` here?

Reply:
It's what's done in the rest of the code. Otherwise, the only alternative is to panic if it's not a valid conversion.